docker/README

```diff
@@ -0,0 +1,20 @@
+These Dockerfiles will create a DronecodeSDK development container image based on whichever flavour of Ubuntu or Fedora you prefer.
```

Comment:
Would be nice to have a markdown file README.md.

Reply:
ok will fix.